test/metabase/api/database_test.clj

```diff
@@ -799,7 +791,17 @@
         (testing "with SSL unspecified, try twice (once with, once without SSL)"
           (#'database-api/test-connection-details "postgres" {})
           (is (= 2 @call-count))
-          (is (= [true false] @ssl-values)))))))
+          (is (= [true nil] @ssl-values)))
```

Comment:
This change is because we no longer `assoc _ :ssl false` but use the original details map. Hope this is fine.